baybe/surrogates/gaussian_process/presets/default.py

```diff
@@ -0,0 +1,110 @@
+"""Default preset for Gaussian process surrogates."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from attrs import define
+
+from baybe.kernels.basic import MaternKernel, ScaleKernel
+from baybe.priors.basic import GammaPrior
+from baybe.surrogates.gaussian_process.kernel_factory import KernelFactory
+
+if TYPE_CHECKING:
+    from torch import Tensor
+
+    from baybe.kernels.base import Kernel
+    from baybe.searchspace.core import SearchSpace
```

Comment:
Yes, yes nothing to do with "bubble" or similar but rather that it's simply enough to import it for type checking. This can be improved pretty much everywhere else in our codebase (together with a list of other import related things I already planned for a short import-PR), but since this code here is new, I directly placed it in the right section. If ok, please resolve